app/components/Exchange/Exchange.jsx

```diff
@@ -316,23 +316,21 @@ class Exchange extends React.Component {
         return this.state.feeStatus[asset.get("id")] && this.state.feeStatus[asset.get("id")].fee;
     }
 
-    _verifyFee(fee, sellAmount, sellBalance, coreBalance) {
+    _verifyFee(fee, sell, sellBalance, coreBalance) {
         let coreFee = this._getFee();
 
-        let sellSum = fee.getAmount() + sellAmount;
-        if (fee.asset_id === "1.3.0") {
+        if (fee.asset_id === "1.3.0") { // Very strange check
```

Comment:
Why do you think this is strange? All fees are eventually paid in BTS, (1.3.0), if they're not it requires special handling to check fee pools etc

Reply:
wouldn't this produce an error if i sell BTS?  there's no check for required amount balance+fee, but only the fee

Reply:
There are checks in the buySell component that disables the submit button in that case. When you click the balance the fee is also already subtracted from the amount you're trying trying to sell.